RNA-Seq analysis with AskOmics interactive tool

xgaia opened the pull request:
Hello everyone,

This is the tutorial to use the [AskOmics IT](https://github.com/galaxyproject/galaxy/pull/9915) in Galaxy.

The tutorial use data from the [RNA-seq counts to genes](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/rna-seq-counts-to-genes/tutorial.html) tutorial.

The AskOmics IT is [not available yet](https://github.com/galaxyproject/galaxy/pull/9915) in Galaxy, but it is available on [galaxy.genouest.org](https://galaxy.genouest.org).

Have a nice day,

Xavier

(ping @abretaud and @bgruening)

hexylena commented on topics/transcriptomics/tutorials/rna-seq-analysis-with-askomics-it/tutorial.bib:
```diff
@@ -0,0 +1,30 @@
+# To cite the example below, in your tutorial.md file
+# use {% Batut2018 %}
+
+@article{Batut2018,
```
Please remove the unused citation

xgaia replied:
Done

xgaia commented:
> You might want to add pre-requirements? Maybe the RNA seq tutorial and add to the RNA seq tuotorial an annotation to look at this tutorial next. This will enable a learning path. See here https://github.com/galaxyproject/training-material/edit/master/topics/transcriptomics/tutorials/ref-based/tutorial.md follow_up_training ... maybe?

I added `rna-seq-counts-to-genes` as a requirement in mh tutorial

ANd I added my tutorial as a follow up training to `rna-seq-counts-to-genes` (plus visualizations tutorials)

bgruening commented:
It should be deployed on EU by now.

bgruening commented:
Thanks @xgaia thanks a lot!!!!
